assignments/models.py

```diff
@@ -121,3 +121,30 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Student(models.Model):
+    assignment = models.ForeignKey(Assignment, related_name='students')
+    school = models.CharField(max_length=256)
+    school_class = models.CharField(max_length=128)
```

Comment:
What is the reason for not having 255 here as well? Is there any 128 max char restriction from the customer?